scenario/BulkerScenario.ts

```diff
@@ -0,0 +1,165 @@
+import { scenario } from './context/CometContext';
+import { constants, utils } from 'ethers';
+import { expect } from 'chai';
+import { isRewardSupported, isBulkerSupported, getExpectedBaseBalance } from './utils';
+import { exp } from '../test/helpers';
+
+scenario(
+  'Comet#bulker > all non-reward actions in one txn',
```

Comment:
I decided to keep this scenario even though we also have another scenario that tests all actions (including claim reward) in one txn. This is because that scenario filters out chains without an initialized rewards contract, so keeping this scenario should still give us Bulker scenario coverage on those chains.